<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RingCoolingSwitchWidget</class>
 <widget class="QWidget" name="RingCoolingSwitchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="lblDevice">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Dev</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QESimpleShape" name="ledFlow">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Error</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(prefix):isFlowInterlock</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>21</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QESimpleShape" name="ledTemperature">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Error</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(prefix):isTemperatureInterlock</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>11</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
